### PR TITLE
Fix ssh

### PR DESCRIPTION
**Pull request by a-gn**
Fix the SSH authentication problem.

### CODE REVIEW

**Comment by a-gn** [comment]
We should do a squash merge since this branch contains a lot of merge commits.

File: diagnosenet/resourcemanager.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def run_device_replica(self, user_name, host_name, device_replica, job_replica):
         Subprocess by device job replica
         """
         #print("++ ssh: {}  || job_replica: {}".format(str(user_name+"@"+host_name), job_replica))
-        sp.call(["ssh", str(user_name+"@"+host_name), "python3.6", device_replica, "{}".format(job_replica)])
+        sp.call(["ssh", "-i", "/home/{}/.ssh/id_rsa".format(user_name), str(user_name+"@"+host_name), "python3.6", device_replica, "{}".format(job_replica)])
```
Comment (jagh) [inline diagnosenet/resourcemanager.py]: To launch the replica node, we left the two modes:
```
if 'astro' in str(host_name):
        ## Astro cluster node replication
        sp.call(["ssh", "-i", "/home/{}/.ssh/id_rsa".format(user_name), str(user_name+"@"+host_name), "python3.6", device_replica, "{}".format(job_replica)])
else:
        ## Rosie cluster node replication
        sp.call(["ssh", str(user_name+"@"+host_name), "python3.6", device_replica, "{}".format(job_replica)])
```

File: samples/0_sequentialgraph/mlp_pmsi_distributed_GRPC_launcher.py [hunk 2]
```diff
@@ -15,9 +15,9 @@
 ## Setting distributed training with resourcemanager:
 distributed_training = ResourceManager()
```
Comment (jagh) [inline samples/0_sequentialgraph/mlp_pmsi_distributed_GRPC_launcher.py:17]: To enable the launcher replica in both clusters, we will add the cluster IPs, as:
```
## Rosie mini-cluster set rol nodes
rosie_ip_ps = "134.59.132.135"
rosie_ip_workers = "134.59.132.20,134.59.132.21,134.59.132.23,134.59.132.26"

## Astro array-server set rol nodes
astro_ip_ps = "134.59.132.185"
astro_ip_workers = "134.59.132.190,134.59.132.192"

## General parameters
num_ps=1
num_workers = 2

## Distributed GRPC training launcher
distributed_training.between_graph_replication(device_replica_path="/home/mpiuser/cloud/0/diagnosenet/samples/0_sequentialgraph/",
                                    device_replica_name="mlp_pmsi_distributed_GRPC_replica.py",
                                    ip_ps=astro_ip_ps, ip_workers=astro_ip_workers,
                                    num_ps=num_ps, num_workers=num_workers)
```

File: samples/0_sequentialgraph/mlp_pmsi_distributed_GRPC_replica.py [hunk 3]
```diff
@@ -7,7 +7,7 @@
 
 ## Makes diagnosenet library visible in samples folder
 import sys
-file_path = "/home/mpiuser/cloud/diagnosenet/"
+file_path = "/home/mpiuser/cloud/0/diagnosenet/"
```
Comment (jagh) [inline samples/0_sequentialgraph/mlp_pmsi_distributed_GRPC_replica.py:10]: ## For the experimentation stage, we left both parameters set for Astro and Rosie platforms:
```
import sys, os

platform = 'astro'
if 'astro' in str(os.uname()[1]):
    file_path = "/home/mpiuser/cloud/0/diagnosenet/"
    testbed_path="/home/mpiuser/cloud/0/diagnosenet/samples/0_sequentialgraph
    dataset_path="/home/mpiuser/cloud/0/diagnosenet/samples/0_sequentialgraph/dataset/"
else: 
    file_path = "/home/mpiuser/cloud/diagnosenet/"
    testbed_path="/home/mpiuser/cloud/diagnosenet/samples/0_sequentialgraph/testbed"
    dataset_path="/home/mpiuser/cloud/diagnosenet/samples/0_sequentialgraph/dataset/"
```

```

**Review by jagh — CHANGES_REQUESTED** [review]

File: diagnosenet/resourcemanager.py (hunk 1 above)
Comment (a-gn) [reply diagnosenet/resourcemanager.py]: This change is on an outdated version of `resourcemanager`, I had already added a condition for astro: https://github.com/IADBproject/diagnosenet/blob/5f2a6e277a2357743ae46a370df92ca746af6087/diagnosenet/resourcemanager.py#L60.

**Pull request merged**